delfin/api/v1/alerts.py

```diff
@@ -61,7 +61,12 @@ def show(self, req, id):
             raise exception.InvalidInput(msg)
 
         storage = db.storage_get(ctx, id)
-        alert_list = self.driver_manager.list_alerts(ctx, id, query_para)
+
+        try:
+            alert_list = self.driver_manager.list_alerts(ctx, id, query_para)
+        except NotImplementedError:
```

Comment:
Why would we wrap NotImplementedError to MethodNotAllowed here

Reply:
maybe ignoring the error is also fine currently